.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,33 @@
+name: Benchmark
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+env:
+  DEFAULT_GO_VERSION: 1.17
+jobs:
+  benchmark:
+    name: Benchmarks
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: ${{ env.DEFAULT_GO_VERSION }}
+      - name: Run benchmarks
+        run: make test-bench | tee output.txt
+      - name: Download previous benchmark data
+        uses: actions/cache@v2
+        with:
+          path: ./benchmarks
+          key: ${{ runner.os }}-benchmark
+      - name: Store benchmarks result
+        uses: benchmark-action/github-action-benchmark@v1
+        with:
+          name: Benchmarks
+          tool: 'go'
+          output-file-path: output.txt
+          external-data-json-path: ./benchmarks/data.json
+          auto-push: false
+          fail-on-alert: true
```

Comment:
Based on [this comment](https://github.com/open-telemetry/opentelemetry-go/pull/2559#issuecomment-1023998664), should this contain alert configuration, or it the 200% deviation the default?

Reply:
Based on https://github.com/benchmark-action/github-action-benchmark#alert-threshold-optional, 200% is the default.